monitoring-proxy/Makefile

```diff
@@ -1,12 +1,9 @@
 .PHONY: build
 
 registryUrl ?= registry.service.opg.digital
-oldRegistryUrl ?= registry.service.dsd.io
 
 image = opguk/monitoring-proxy
 
 build:
 	docker build $(no-cache) -t "$(registryUrl)/$(image)" .
 	docker tag -f "$(registryUrl)/$(image)" "$(registryUrl)/$(image):$(newtag)"
```

Comment:
can we not ditch the -f now on the docker-tag as well, it's deprecated and fails in later versions